data/json/monsters/insect_spider.json

```
@@ -400,37 +399,116 @@
+    "default_faction": "aquatic_predator",
+    "aggression": 40,
+    "morale": 25,
+    "melee_skill": 6,
+    "melee_dice": 1,
+    "melee_dice_sides": 5,
+    "melee_cut": 5,
+    "special_attacks": [ [ "STRETCH_ATTACK", 25 ] ],
+    "armor_cut": 6,
+    "armor_bullet": 5,
+    "dodge": 3,
+    "vision_day": 25,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "fear_triggers": [ "HURT" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "AQUATIC", "ARTHROPOD_BLOOD", "NO_BREATHE", "SEES", "SWIMS" ],
+    "upgrades": { "age_grow": 24, "into": "mon_dragonfly_giant" }
+  },
   {
     "id": "mon_dragonfly_giant",
     "type": "MONSTER",
     "name": { "str": "giant dragonfly", "str_pl": "giant dragonflies" },
-    "description": "A ferocious mutant dragonfly as big as a cat, darting through the air with a cluster of fangs for a mouth.",
+    "description": "A ferocious, bright green dragonfly, still lightning-fast despite growing to the size of a schoolchild.  Its head is little more than a basketball-sized pair of rainbow compound eyes, and its legs - already specialized for grasping prey - end in curved talons.",
     "default_faction": "dragonfly",
     "bodytype": "flying insect",
-    "species": [ "INSECT" ],
-    "volume": "30 L",
+    "species": [ "INSECT_FLYING" ],
+    "volume": "40 L",
     "weight": "40750 g",
-    "hp": 70,
+    "hp": 50,
     "speed": 150,
     "material": [ "iflesh" ],
     "symbol": "y",
     "color": "light_green",
     "aggression": 20,
-    "morale": 100,
-    "melee_skill": 6,
+    "morale": 10,
+    "melee_skill": 5,
     "melee_dice": 1,
     "melee_dice_sides": 6,
-    "melee_cut": 6,
-    "dodge": 8,
-    "armor_cut": 6,
-    "armor_bullet": 5,
+    "melee_damage": [ { "damage_type": "cut", "amount": 8, "armor_penetration": 10 } ],
+    "special_attacks": [
+      [ "GRAB", 100 ],
+      {
+        "type": "bite",
+        "cooldown": 6,
+        "accuracy": 5,
+        "move_cost": 300,
+        "damage_max_instance": [ { "damage_type": "cut", "amount": 6, "armor_penetration": 20 } ]
+      }
+    ],
+    "dodge": 2,
+    "vision_day": 45,
+    "vision_night": 5,
+    "harvest": "arachnid_flying",
+    "anger_triggers": [ "PLAYER_WEAK", "STALK" ],
+    "fear_triggers": [ "HURT" ],
+    "death_function": [ "NORMAL" ],
+    "upgrades": { "age_grow": 40, "into": "mon_dragonfly_mega" },
+    "flags": [ "SWARMS", "HEARS", "SEES", "FLIES", "PATH_AVOID_FIRE", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_dragonfly_mega",
+    "type": "MONSTER",
+    "name": { "str": "mega dragonfly", "str_pl": "mega dragonflies" },
+    "description": "A single, enormous insect with a swollen abdomen and a distended ovipositor, periodically dropping a handful of glistening eggs.",
+    "volume": "600 L",
+    "weight": "800 kg",
+    "default_faction": "dragonfly",
+    "bodytype": "insect",
+    "species": [ "INSECT_FLYING" ],
+    "hp": 1100,
```

I just now however noticed that is more than twice the HP of hulks, dafuq? Especially when the PR already nerfs regular giant dragonflies down from 70 to 50 HP.
```suggestion
    "hp": 110,
```
I assume this was an accident and it's off by a factor of 10.